instrumentation-api/src/main/java/io/opentelemetry/instrumentation/api/instrumenter/NeverSuppressAndStoreStrategy.java

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.instrumentation.api.instrumenter;
+
+import io.opentelemetry.api.trace.Span;
+import io.opentelemetry.api.trace.SpanKind;
+import io.opentelemetry.context.Context;
+import java.util.List;
+
+final class NeverSuppressAndStoreStrategy extends SpanSuppressionStrategy {
```

Comment:
Nit: I read this with `Never` applying to `Store` too. `StoreOnly` would make sense to me